### ISSUE TITLE
Pencil icon on the dropdown from query selection in a form does not work

### ISSUE THREAD

**Issue by eXistenZe** [2sic/2sxc#1761]
**I'm submitting a ...** 
[x] bug report

**...about**
[x] Content Types or data management

**Current behavior**
When you are creating or editing a record that contains a dropdown from query field, it shows a dropdown icon that shows the options to select the values/labels from the query, and a pencil icon.

Not sure what this is supposed to do, since it comes from a query and not from an entity, but it just shows a greyed full screen div and nothing happens.

Screen: https://pasteboard.co/IhfZ7vp.jpg
Browser log:

```
GET https://site.pt/desktopmodules/tosic_sexycontent/dist/ng-edit/i18n/pt.js 404 (Not Found)
ERROR e {headers: t, status: 404, statusText: "Not Found", url: "https://site.pt/desktopmodules/tosic_sexycontent/dist/ng-edit/i18n/pt.js", ok: false, …}
POST https://site.pt/pt-pt/desktopmodules/2sxc/api/eav/ui/load?appId=97 400 (Bad Request)
Http failure response for https://site.pt/pt-pt/desktopmodules/2sxc/api/eav/ui/load?appId=97: 400 Bad Request
ERROR Http failure response for https://investigar.pt/pt-pt/desktopmodules/2sxc/api/eav/ui/load?appId=97: 400 Bad Request
```

**Expected behavior**
Unsure

* **2sxc version(s):** 10

**Comment by iJungleboy** [comment]
I think we just fixed this in 10.01 released a few minutes ago. Can you confirm? If it's not fixed (should be), we'll reopen this.

**Comment by eXistenZe** [comment]
Nope. Same thing.

**Comment by iJungleboy** [comment]
ok, @PagnoDunadan will have a look.

**Comment by iJungleboy** [comment]
fixed in 10.02
